samples/csharp_dotnetcore/07.using-adaptive-cards/readme.md

```diff
@@ -23,22 +23,48 @@ git clone https://github.com/microsoft/botbuilder-samples.git
 - Hit F5
 ## Visual Studio Code
 - Open `botbuilder-samples/samples/csharp_dotnetcore/07.using-adaptive-cards` folder
-- Bring up a terminal, navigate to botbuilder-samples/samples/csharp_dotnetcore/07.using-adaptive-cards
-- Type 'dotnet run'.
+- Bring up a terminal, navigate to `botbuilder-samples/samples/csharp_dotnetcore/07.using-adaptive-cards`
+- Type `dotnet run`.
 ## Update packages
 - In Visual Studio right click on the solution and select "Restore NuGet Packages".
+- In Visual Studio Code type `dotnet restore`
+# Use Azure Blob Storage or Azure CosmosDB
+
+##### In Visual Studio
```

Comment:
I don't think we need the Azure Blog Storage section in here. This sample is focused on Adaptive card, and adding in the additional text makes things more confusing.

Reply:
We removed the `Azure Blob Storage` section from the Readme.